### PR TITLE
Introduce kolla_copy_cacerts

### PR DESCRIPTION
**Pull request by stuggi**
This adds a script to run at container start via kolla_start to update the container environment for trusted CAs (TLS-E).

Also adds the cloud-admin user to the kolla group to be allowed to run kolla* commands as root. This is required to get the openstackclient CA trust updated for tls endpoints.

Jira: [OSP-26299](https://issues.redhat.com//browse/OSP-26299)
Jira: [OSP-26849](https://issues.redhat.com//browse/OSP-26849)

### CODE REVIEW

**Comment by stuggi** [comment]
recheck

File: container-images/kolla/base/start.sh [hunk 1]
```diff
@@ -9,6 +9,9 @@ sudo -E kolla_set_configs
 CMD=$(cat /run_command)
 ARGS=""
 
+# Install custom CA certificates
+sudo kolla_update_cacerts
```
Comment (stuggi) [inline container-images/kolla/base/start.sh:13]: not all users are in kolla group and can run this via sudo

Comment (stuggi) [reply container-images/kolla/base/start.sh:13]: well `sudo -E kolla_set_configs` above is also good, so we can expect this to work

**Comment by stuggi** [comment]
not sure why cinder fails:
```
2023-09-29 04:16:01 | Stderr: 'Error: building at STEP "RUN dnf -y install ceph-common openstack-cinder && dnf clean all && rm -rf /var/cache/dnf": while running runtime: exit status 1\n': RuntimeError: [00m
```

**Comment by raukadah** [comment]
https://logserver.rdoproject.org/82/82/3ad878147c64a0f50e17977ae3dda609d989f349/github-check/tcib-build-containers/08121fb/ci-framework-data/logs/750e6bb7-bc67-42d9-af49-4cd6d7ca3ac8/base/os/cinder-base/cinder-base-build.log
```
Extra Packages for Enterprise Linux 9 - x86_64   51 MB/s |  19 MB     00:00    
Error: 
 Problem: package python3-grpcio-1.41.1-6.el9s.x86_64 from delorean-antelope-testing requires libre2.so.0a()(64bit), but none of the providers can be installed
  - package python3-google-api-core-2.10.2-1.el9s.noarch from delorean-antelope-testing requires python3dist(grpcio), but none of the providers can be installed
  - cannot install both re2-1:20211101-3.el9.x86_64 from epel-low-priority and re2-1:20190801-9.el9s.x86_64 from delorean-antelope-testing
  - cannot install both re2-1:20211101-3.el9.x86_64 from epel-low-priority and re2-1:20190801-9.el9s.x86_64 from centos9-storage
  - package python3-google-api-client-2:2.58.0-1.el9s.noarch from delorean-antelope-testing requires ((python3.9dist(google-api-core) < 2 or python3.9dist(google-api-core) > 2.1) with (python3.9dist(google-api-core) < 2.1 or python3.9dist(google-api-core) > 2.2) with (python3.9dist(google-api-core) < 2.2 or python3.9dist(google-api-core) > 2.3) with (python3.9dist(google-api-core) < 2.3 or python3.9dist(google-api-core) > 2.3) with python3.9dist(google-api-core) < 3~~dev0 with python3.9dist(google-api-core) >= 1.31.5), but none of the providers can be installed
  - package libarrow-9.0.0-9.el9.x86_64 from epel-low-priority requires libre2.so.9()(64bit), but none of the providers can be installed
  - package python3-cinder-1:22.1.1-0.20230914070759.bdf0a3d.el9.noarch from delorean-component-cinder requires python3-google-api-client >= 1.11.0, but none of the providers can be installed
  - package parquet-libs-9.0.0-9.el9.x86_64 from epel-low-priority requires libarrow(x86-64) = 9.0.0-9.el9, but none of the providers can be installed
  - package openstack-cinder-1:22.1.1-0.20230914070759.bdf0a3d.el9.noarch from delorean-component-cinder requires python3-cinder = 1:22.1.1-0.20230914070759.bdf0a3d.el9, but none of the providers can be installed
  - package ceph-common-2:18.2.0-1.el9s.x86_64 from centos9-storage requires libparquet.so.900()(64bit), but none of the providers can be installed
  - conflicting requests
  - nothing provides libthrift-0.14.0.so()(64bit) needed by parquet-libs-9.0.0-1.el9s.x86_64 from centos9-storage
  - package python3-grpcio-1.46.7-10.el9.x86_64 from epel-low-priority is filtered out by exclude filtering
(try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages)
```
It appears to be missing package issue.

**Comment by raukadah** [comment]
recheck

File: container-images/kolla/base/start.sh (hunk 1 above)
Comment (olliewalsh) [reply container-images/kolla/base/start.sh:13]: it's `kolla_copy_cacerts`  in kolla, should we try to keep it consistent?

Comment (stuggi) [reply container-images/kolla/base/start.sh:13]: we could, thought we are now disconnected from kolla.

Comment (olliewalsh) [reply container-images/kolla/base/start.sh]: yeah but likely to hit similar issues in both, with similar fixes, so good to stay as consistent as possible I expect

**Review by olliewalsh — APPROVED** [review]
/lgtm

**Comment by stuggi** [comment]
@raukadah could you approve this?

**Pull request merged**